### PR TITLE
Bug 2018449: add test for network attachment definition

### PR DESCRIPTION
**Pull request by gouyang**

### CODE REVIEW

**Comment by gouyang** [comment]
@yaacov @pcbailey request review.

File: frontend/packages/kubevirt-plugin/integration-tests-cypress/support/index.ts [hunk 1]
```diff
@@ -28,6 +28,7 @@ declare global {
       selectProject(project: string): void;
       createNAD(namespace: string): void;
       waitForLoginPrompt(vmName: string, namespace: string): void;
+      visitNAD(): void;
```
Comment (pcbailey) [inline frontend/packages/kubevirt-plugin/integration-tests-cypress/support/index.ts:31]: I think `visitNADPage` would be a better/clearer name, but I'm not going to hold up this PR for that.

**Pull request merged**